src/main/java/exceptions/DukeBlankDetailsException.java

```diff
@@ -0,0 +1,13 @@
+package exceptions;
+
+public class DukeBlankDetailsException extends Exception {
+
+    /**
+     * Added default serial versio uid
+     */
+    private static final long serialVersionUID = 1L;
```

Comment:
For names of constants, maybe capitalizing and using underscores to separate words would be better?